web/app.js

```diff
@@ -811,8 +811,7 @@ const PDFViewerApplication = {
    */
   _hideViewBookmark() {
     // URL does not reflect proper document location - hiding some buttons.
-    const { toolbar, secondaryToolbar } = this.appConfig;
-    toolbar.viewBookmark.hidden = true;
+    const { secondaryToolbar } = this.appConfig;
     secondaryToolbar.viewBookmarkButton.hidden = true;
```

Comment:
With the changes here, we could just directly do the following instead:
```suggestion
    this.appConfig.secondaryToolbar.viewBookmarkButton.hidden = true;
```